Sources/PCCurrentUser.swift

```diff
@@ -554,7 +554,7 @@ public final class PCCurrentUser {
 
     // MARK: Message-related interactions
 
-    func sendMessage(_ messageObject: [String: Any], roomID: String, completionHandler: @escaping (Int?, Error?) -> Void) {
+    func sendMessage(instance: Instance, _ messageObject: [String: Any], roomID: String, completionHandler: @escaping (Int?, Error?) -> Void) {
```

Comment:
It weirds me out that this method takes `[String: Any]` when we have perfectly good message types, but that's probably a refactor for another day... 😛

Reply:
Ah yes! I know..I'm keeping it consistent with what is already there :P The library could do with a refactor for sure.